### PR TITLE
Fix taxonomy filters selected tag color and radio button checked state

### PR DESCRIPTION
**Pull request by mbice**
Fix selected filter tag text color and ensure checked state of radio button is applied. Applies to #3421.

### CODE REVIEW

File: search-parts/src/components/filters/FilterSearchBoxComponent.module.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
     max-width: 100%;
 
     .tagItemText {
+        color: #fff;
```
Comment (wobba) [inline search-parts/src/components/filters/FilterSearchBoxComponent.module.scss:20]: Should this use the theme color for white instead? Which would then be black on high contast.

Comment (mbice) [reply search-parts/src/components/filters/FilterSearchBoxComponent.module.scss]: @wobba good point. I updated it now to use the theme, instead. Thanks!

Comment (mbice) [reply search-parts/src/components/filters/FilterSearchBoxComponent.module.scss]: @wobba sorry, one other thing. If this PR is good and gets merged in, do you think there would be any chance to get a 4.10.2 patch release?

**Comment by wobba** [comment]
Thank you

File: search-parts/src/components/filters/FilterSearchBoxComponent.module.scss (hunk 1 above)
Comment (LeonArmston) [reply search-parts/src/components/filters/FilterSearchBoxComponent.module.scss]: Great work @mbice - I'd love this in a 4.10.2 release - I've got a client this is affecting at the moment.

**Pull request merged**